contract/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "hardhat-project",
+  "scripts": {
+    "test": "npm run prettier && npx hardhat compile && npx hardhat test",
+    "clean": "npx hardhat clean",
+    "coverage": "npx hardhat coverage && cat coverage/lcov.info | coveralls",
+    "compile": "npx hardhat compile",
+    "deploy": "npx hardhat deploy",
+    "prettier:solidity": "npx prettier --write contracts/.",
+    "prettier:typescript": "npx prettier --write test/.",
+    "prettier": "npx prettier --write contracts/. test/.",
+    "lint": "eslint . --max-warnings 0 --ext .ts"
+  },
+  "devDependencies": {
+    "@nomicfoundation/hardhat-toolbox": "^2.0.2",
+    "coveralls": "^3.1.1",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "hardhat": "^2.14.1",
+    "hardhat-deploy": "^0.11.30",
+    "lcov2badge": "^0.1.2",
+    "prettier": "^2.8.8",
```

Comment:
eslint and prettier seem to conflict a lot of times. I usually just use eslint and configure prettier to process .sol files only